### PR TITLE
Update getting started with TF remote & how TF fits into the HashiCorp ecosystem

### PR DESCRIPTION
**Pull request by KFishner**
cc @mitchellh @phinze 

note that i'll deploy the atlas mindset blog post first

### CODE REVIEW

**Comment by phinze** [comment]
Gave it a once-over - LGTM!

File: website/source/intro/getting-started/remote.html.markdown [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+We've now seen how to build, change, and destroy infrastructure
+from a local machine. This is great for testing and development,
+however in production environments it is more responsible to run
+Terraform remotely and store a master Terraform state remotely.
+Otherwise it's possible for multiple different Terraform states
+to be stored on developer machines, which could lead to conflicts.
+Additionally by running Terraform remotely, you can move access
+credentials off of developer machines, release local machines from
+long-running Terraform processes, and store a history of
+infrastructure changes to help with auditing and collaboration.
```
Comment (mitchellh) [inline website/source/intro/getting-started/remote.html.markdown]: I think this doesn't hit upon the biggest benefit: collaboration and working in a multi-team org

**Comment by pearkes** [comment]
It might be worth pointing them to the [the tutorial](https://atlas.hashicorp.com/tutorial/terraform) as it handholds through mostly the same process!

**Comment by KFishner** [comment]
good call @pearkes! 

@mitchellh i'll make collab more prominent

**Comment by mitchellh** [comment]
LGTM

**Pull request merged**